Source/WebCore/Modules/webxr/WebXRSession.cpp

```diff
@@ -139,7 +139,7 @@ ExceptionOr<void> WebXRSession::updateRenderState(const XRRenderStateInit& newSt
 
     // 3. If newState's baseLayer was created with an XRSession other than session,
     //    throw an InvalidStateError and abort these steps.
-    if (newState.baseLayer && newState.baseLayer->session() != this)
+    if (newState.baseLayer.has_value() && newState.baseLayer.value() && (*newState.baseLayer)->session() != this)
```

Comment:
I find this easier to read as:

```suggestion
    if (newState.baseLayer && newState.baseLayer.value() && newState.baseLayer.value()->session() != this)
```